### ISSUE TITLE
Current and Next Version numbers should be asked for at the start of the build.

### ISSUE THREAD

**Issue by swpalmer** [researchgate/gradle-release#182]
Some build steps may take a long time. It's frustrating to start a release build and walk away because it starts a long running task, only to come back and see that it has been stalled because it is prompting to confirm the current version number or asking what the next version number is for a project.
These questions should be asked before other build tasks begin.

**Comment by Morl99** [comment]
I want to second this. This is causing problems in many of our projects, as sometimes developers forget to push enter a second time, leaving the project in an unsnapshoted state. 
I would really like to control automaticVersioning in gradle, but I cannot find a way to do this, as it seems like this can only be done through an environment variable (which I cannot/do not want to do). If I could put this into the gradle.properties file, it would be ideal.
But the enhancment that is proposed by @swpalmer  would solve the issue as well.

**Comment by Hillkorn** [comment]
But you can do exactly that. Tried with gradle 2.14.1 in https://github.com/Hillkorn/release-bug
I've put into the gradle.properties
release.useAutomaticVersion=true

**Comment by swpalmer** [comment]
Yes, release.useAutomaticVersion=true  is a partial workaround.  However, it doesn't address the issue for when you want to be prompted but the prompts don't appear until well into the build process.
